__tests__/models.test.js

```diff
@@ -129,3 +129,40 @@ describe.only('GET/api/articles', () => {
   })
 })
 
+describe('GET/api/articles/:article_id/comments', () => {
+  it('200: returns an array of comments for article id requested by user sorted by newest comment first', () => {
+    return request(app)
+    .get('/api/articles/1/comments')
+    .expect(200)
+    .then(({body}) => {
+      const {comments} = body
+      expect(comments).toBeSortedBy('created_at', {descending: true})
+      expect(comments).toHaveLength(11)
+      
+      comments.forEach(comment => {
+        expect(comment).toHaveProperty('votes', expect.any(Number))
+        expect(comment).toHaveProperty('body', expect.any(String))
+        expect(comment).toHaveProperty('author', expect.any(String))
+        expect(comment).toHaveProperty('comment_id', expect.any(Number))
+        expect(comment).toHaveProperty('article_id', expect.any(Number))
+        expect(comment).toHaveProperty('created_at', expect.any(String))
+      })
+    })
```

Comment:
Another status 200 test would be nice here 👀 you could check if the article ID exists but the comments array is empty 🤸